docs/GettingStarted.md

```diff
@@ -81,31 +81,65 @@ jest --init
 
 ### Using Babel
 
-To use [Babel](http://babeljs.io/), install the `babel-jest` and `@babel/core` packages via `yarn`:
+To use [Babel](http://babeljs.io/), install required dependencies via `yarn`:
 
 ```bash
-yarn add --dev babel-jest @babel/core
+yarn add --dev babel-jest @babel/core @babel/preset-env @babel/preset-react
```

Comment:
you don't _need_ either of the presets - the other 2 are actually needed, though. Not sure how to best say that, and still have useful examples

Reply:
Well you'd probably want to include something, at the very least you probably want to compile ES6 modules.

I think our bases are pretty well covered by "The ideal configuration for Babel will depend on your project.".

Reply:
We assume user already have Babel set up in the project and they only want to add Jest support, hence only `babel-jest` and `@babel/core` are needed. They need to be at top level of `node_modules` to satisfy peerDependencies and transform resolver

Reply:
for babel 7, people are required to use `@babel/core`. So if we go with the assumption `babel` is correctly set up, we just need to add `babel-jest`

Reply:
I think it's sane to assume that Babel is there and we should make it obvious in the guide. We can't choose Babel plugins and presets for users, that's bad.

So, only adding `babel-jest` should suffice. We just need to make sure users have Babel 7 already set up.